build.install4j

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <install4j version="6.1.2" transformSequenceNumber="5">
-  <directoryPresets config="tutorial.zip" />
+  <directoryPresets config="./game_engine.properties" />
```

Comment:
I have no idea how this element is used by install4j, and I couldn't find any way to change it from within the install4j application.  So I simply reverted it to the value it had prior to #1492, when it was last changed.